laske_export/tests/test_payments.py

```diff
@@ -0,0 +1,81 @@
+import os
+import tempfile
+
+import pytest
+from django.conf import settings
+
+from laske_export.management.commands import get_payments_from_laske
+
+
+@pytest.mark.django_db
+def test_unimported_files(monkeypatch):
+    test_files = [
+        "MR_OUT_ID256_8000_20190923_014512.TXT",
+        "MR_OUT_ID256_8000_20191002_014527.TXT",
+        "MR_OUT_ID256_8000_20191011_014517.TXT",
+    ]
+    test = get_payments_from_laske.Command()
```

Comment:
Better naming could help the reader (me) to understand what is happening here. Maybe something like

```suggestion
    laske_command = get_payments_from_laske.Command()
```